cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporter.java

```diff
@@ -85,30 +86,36 @@ public void close() {
 
   @Override
   public void configure(Map<String, ?> configs) {
-    String bootstrapServers = (String) configs.get(CRUISE_CONTROL_METRICS_REPORTER_BOOTSTRAP_SERVERS);
-    if (bootstrapServers == null) {
+    Properties producerProps = CruiseControlMetricsReporterConfigs.parseConfigMap(configs);
```

Comment:
If CruiseControlMetricsRepoterConfigs just extended from the Kafka configs class it would be easy to put all this default logic into that class.  No?